src/App.vue

```diff
@@ -25,8 +25,11 @@
                 @create="createTimeZone"
               ></AddTimeZoneModal>
             </div>
+
           </div>
 
+          <va-slider v-model="sliderValue" :min="1" :max="24" track-label-visible/>
```

Comment:
Let's set offset from -12 to 12 and value 0 by default.

```suggestion
          <va-slider v-model="sliderValue" :min="-12" :max="12" track-label-visible/>
```